### ISSUE TITLE
Travis compile

### ISSUE THREAD

**Issue by RMcGirr83** [travis-ci/travis-ci#5217]
Unable to locate package parallel

**Comment by BanzaiMan** [comment]
Could you try running `sudo apt-get update` beforehand?

**Comment by RMcGirr83** [comment]
Same message

https://travis-ci.org/RMcGirr83/phpBB-3.1-nationalflags/jobs/95060956

**Comment by BanzaiMan** [comment]
Do you have the APT source that has this package?

**Comment by RMcGirr83** [comment]
Not sure what you are asking.  All I know is since the travis update none of my extensions for phpBB are completing now and they all reference the same error.  I am not sure if it is because of this or not

"This job ran on our new platform for Legacy Precise builds. Please read our blog post for more information."

but there seems to be a correlation.

Read the blog post but it's a bit greek to me.

**Comment by RMcGirr83** [comment]
After further review, if I set PHP version 5.3.3 to be an allowed failure eveything works, and passes, travis just fine.  So it would seem to be an issue with your current implementation @BanzaiMan.  For reference https://github.com/LMOspaceace/ajaxchat/blob/master/.travis.yml#L27

As I stated previously, everything worked fine prior to the latest implementation.  So something in the current broke bc.
